filters/f_auto_filters.c

```diff
@@ -242,3 +242,89 @@ struct mp_filter *mp_autorotate_create(struct mp_filter *parent)
 
     return f;
 }
+
+struct aspeed_priv {
+    struct mp_subfilter sub;
+    double cur_speed;
+};
+
+static void aspeed_process(struct mp_filter *f)
+{
+    struct aspeed_priv *p = f->priv;
+
+    if (!mp_subfilter_read(&p->sub))
+        return;
+
+    // (we never remove the filter, because of some old semantics)
```

Comment:
Well, the old code doesn't do this either, so I preferred to keep this for now. I have no idea if it's lossless at 1.0. Should I change it?

Reply:
I think it would be best to remove the filter at 1.0 whether or not it's lossless. I guess one challenge is determining what counts for 1.0. Would we really require floating point equality or just equal to 1.0 within 1e-15 or something.